public/index.html

```diff
@@ -52,11 +52,6 @@
       })(window,document,'script','dataLayer','GTM-TMMV2DF');
     </script>
     <!-- End Google Tag Manager -->
-    <script
```

Comment:
Please leave this as it is. We also use this to track, this is not related only to push.

Reply:
Good catch! Thanks!

Reply:
Done!